lib/autoproj/ops/build.rb

```diff
@@ -84,7 +86,26 @@ def force_build_packages(selected_packages, all_enabled_packages)
             def build_packages(all_enabled_packages, options = Hash.new)
                 Autobuild.do_rebuild = false
                 Autobuild.do_forced_build = false
-                Autobuild.apply(all_enabled_packages, "autoproj-build", ['build'], options)
+                begin
+                    Autobuild.apply(all_enabled_packages, "autoproj-build", ['build'], options)
+                ensure
+                    build_report(all_enabled_packages)
+                end
+            end
+
+            TEMPLATES_DIR = File.join(File.dirname(__dir__), 'templates')
+
+            def build_report(package_list)
+                Autobuild.message "INFO: creating build report...."
```

Comment:
Autobuild/Autoproj don't prefix their messages with INFO and stuff. Moreover, this would be a debug message - we usually don't emit log messages for this kind of internal processes (otherwise we would get twice as much output)